proxy/http/remap/PluginFactory.cc

```diff
@@ -76,6 +76,18 @@ RemapPluginInst::doRemap(TSHttpTxn rh, TSRemapRequestInfo *rri)
   return _plugin.doRemap(_instance, rh, rri);
 }
 
+std::shared_ptr<TSNextHopSelectionStrategy>
+RemapPluginInst::getStrategy()
```

Comment:
There seems a bit of a name semantic mismatch - the outer is "get" but the internal is "init". Does this initialize or just get?

Reply:
This gets the strategy that the func it calls initializes. It's conceptually debatable, I don't object to renaming it

Reply:
Done